sapmachine-11/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM sapmachine:11
+
+ARG MAVEN_VERSION=3.8.4
+ARG USER_HOME_DIR="/root"
+ARG SHA=a9b2d825eacf2e771ed5d6b0e01398589ac1bfa4171f36154d1b5787879605507802f699da6f7cfc80732a5282fd31b28e4cd6052338cbef0fa1358b48a5e3c8
+ARG BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
+
+RUN apt-get update \
+    && apt-get install -y curl \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
You might want to move this up so it's consistent with the other images and version bumps of mvn do not affect the installation of dependencies

Reply:
yes, it would be better higher in the dockerfile

Reply:
I've moved it accordingly